libs/shared/problem/src/ProblemType.ts

```diff
@@ -0,0 +1,8 @@
+export enum ProblemType {
+  HTTP_BAD_REQUEST = 'https://httpstatuses.com/400',
+  HTTP_UNAUTHORIZED = 'https://httpstatuses.com/401',
+  HTTP_FORBIDDEN = 'https://httpstatuses.com/403',
+  HTTP_NOT_FOUND = 'https://httpstatuses.com/404',
+  HTTP_INTERNAL_SERVER_ERROR = 'https://httpstatuses.com/500',
+  VALIDATION_PROBLEM = 'https://docs.devland.is/reference/problems/validation-problem',
```

Comment:
I might have missed it, but where are you going to define this problem in the docs?

Reply:
It's also missing from the readme, that is, how to add the problem information to docs.devland.is for dereferencing